### DISCUSSION TITLE
Mask not formatting numeric value

### DISCUSSION THREAD

**Discussion by GhostyJade** [uNmAnNeR/imaskjs#783] [Q&A]
Hi. I'm trying to format a value that has commas and dots in it (by copy-pasting, eg: 500.000,12).
The expected result should be the copied value, but the mask returns it as 500
What am I doing wrong.

I'm using react-imask@6.4.2 and the mask is the following:
``` javascript
    const mask = useIMask({
	mask: Number,
	scale: 2,
	thousandsSeparator: ' ',
	radix: ',',
	mapToRadix: [',', '.'],
	normalizeZeros: true	
    });
```

**Comment by mengshuqiang** [comment]
https://github.com/uNmAnNeR/imaskjs/issues/847
